library-valcon-internship/src/services/BookService.ts

```diff
@@ -54,3 +55,17 @@ export const getBooks = async ({ pageNumber, pageLength, search, filter, sort }:
 export const addNewBook = async (formData: FormData) => {
   return axios.post(baseUrl + 'api/Books', formData)
 }
+
+export const getBookById = async (id: string | undefined) => {
```

Comment:
why are we receiving undefined to get a book, shouldn't this be mandatory parameter?